examples/maps/maps.bal

```diff
@@ -93,7 +93,7 @@ public function main() {
     // all the fields of the relevant mapping value are added to the new
     // mapping value being created.
     // A spread field is used with `modifiedMarks` to include all the entries
-    // in `modifiedMarks` when creating `allMarks`.
-    map<int> allMarks = {jane: 100, ...modifiedMarks, amy: 75};
-    io:println(allMarks);
+    // in `modifiedMarks` when creating `newMarks`.
+    map<int> newMarks = { ...modifiedMarks};
```

Comment:
What's the need for this change?

Reply:
This is related to https://github.com/ballerina-platform/ballerina-lang/issues/22504. 

```
It is a compile-time error if the static type of the expression in a spread-field 
allows a field that duplicates a specific-field
``` 
(from spec)

Reply:
I see. But then, now the example itself seems sort of broken doesn't it? :thinking:

Reply:
Yeah, I think we need to rewrite this example. Shall we demonstrate this with two separate closed records?

Reply:
Used one closed record and 'cloneWithType'